platform/src/main/java/org/stellar/anchor/server/configurator/PropertiesReader.java

```diff
@@ -0,0 +1,61 @@
+package org.stellar.anchor.server.configurator;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import lombok.SneakyThrows;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.boot.env.YamlPropertySourceLoader;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.env.MapPropertySource;
+import org.springframework.core.env.PropertySource;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.Resource;
+
+public class PropertiesReader extends AbstractConfigurator
+    implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+
+  @SneakyThrows
+  @Override
+  public void initialize(@NotNull ConfigurableApplicationContext applicationContext) {
+    loadConfigYaml("default", new ClassPathResource("default-config.yaml"));
+
+    // Read from $USER_HOME/.anchor/anchor-config.yaml
+    File yamlFile = new File(System.getProperty("user.home"), "./anchor/anchor-config.yaml");
+    if (yamlFile.exists()) {
+      loadConfigYaml("yaml", new FileSystemResource(yamlFile));
+      return;
+    }
```

Comment:
Wait, shouldn't this be the last option according to the documentation?

Reply:
Good catch. I was doing property overloading based on order of precedence then changed to load one of them. I forgot to reverse the order.

Fixed.